core/ir/riscv64/instr.c

```diff
@@ -58,7 +58,7 @@ opc_is_not_a_real_memory_load(int opc)
 {
     /* FIXME i#3544: Not implemented */
     ASSERT_NOT_IMPLEMENTED(false);
```

Comment:
Do we still need this ASSERT_NOT_IMPLEMENTED?

Reply:
I would prefer to leave it. I have not verified whether it will be the only instruction to match. I was more focused on the decoder itself.